Scripts/Utilities/MaterialUtils.cs

```diff
@@ -20,8 +20,12 @@ static class MaterialUtils
         /// <returns>Cloned material</returns>
         public static UnityMaterial GetMaterialClone(Renderer renderer)
         {
+#if UNITY_EDITOR
             // The following is equivalent to renderer.material, but gets rid of the error messages in edit mode
             return renderer.material = UnityObject.Instantiate(renderer.sharedMaterial);
+#else
+            return renderer.material;
```

Comment:
Any reason that instantiating the sharedMaterial as above is not appropriate?

Reply:
Unity copies on access by default for renderer.material. Or I might have gotten a warning with the the instantiation of the shared material when in play  mode. Not sure.

Reply:
This change makes sense to me. This method exists to avoid the warning about using renderer.material in edit mode. In play mode, we can use this functionality without issue.